modules/google-analytics/config/googleAnalytics/ID.ts

```diff
@@ -0,0 +1,2 @@
+export default null
+throw new Error('Override Google Analytics ID')
```

Comment:
Change message to Override config/googleAnalytics/ID so user immediately knows what to override.